src/solver/conv_asm_1x1u_stride2.cpp

```diff
@@ -99,6 +99,14 @@ inline static bool IsLinear(const int v)
     return L <= v && v <= H;
 }
 
+static inline size_t divide_round_plus_inf(const size_t x, const unsigned y)
+{
+    assert(x >= 0 && y > 0);
```

Comment:
@atamazov cppcheck issue in the CI
```
[2023-01-11T01:22:45.356Z] /home/jenkins/workspace/MLLibs_MIOpen_PR-1906/src/solver/conv_asm_1x1u_stride2.cpp:104:14: warning: Unsigned expression 'x' can't be negative so it is unnecessary to test it. [unsignedPositive]
[2023-01-11T01:22:45.356Z]     assert(x >= 0 && y > 0);
[2023-01-11T01:22:45.356Z]              ^

Reply:
@junliume Thanks. Will fix when PR is ready for review.